qiskit/circuit/quantumcircuit.py

```diff
@@ -965,6 +965,45 @@ class QuantumCircuit:
     .. automethod:: qubit_start_time
     .. automethod:: qubit_stop_time
 
+
+    .. _circuit-abstract-physical:
+
+    Abstract and physical circuits
+    ==============================
+
+    Circuits are a fairly low-level abstraction of quantum algorithms.  However, even within this,
+    there are distinctions. Quantum programmers often want to use a wide arrange of gates and
+    instructions, and work in a regime where all qubits and interact with all others.  Quantum
+    hardware, however, typically has a restrictive set of native gates, and only certain pairs of
+    hardware qubits can interact.  We term these two regimes "abstract circuits" and "physical
```

Comment:
Pretty much each sentence starts with a double space after the period I thought we weren't supposed to do that in the docs. It's not like we're using typewriters for this.

```suggestion
    hardware qubits can interact.  We term these two regimes "abstract circuits" and "physical
```

Reply:
This is unfortunately a bit of a lost cause with me - at some point I just gained the muscle memory that that's how I type, and it's proven pretty impossible to shake. You'll probably find that every bit of pride I've ever written for Qiskit is like that too.

Tbf, any competent text renderer shouldn't care about the extra whitespace.